protocol/simplex-messaging.md

```diff
@@ -98,7 +101,7 @@ encoded = <base64 encoded>
 
 `port` is optional, the default TCP port for SMP protocol is 5223.
 
-`serverKeyHash` is an optional hash of the server transport key used during transport handshake (see [Appendix A](#appendix-a)).
+`serverIdentity` is a required hash of the server certificate SPKI (without line breaks) used by the client to validate server certificate during transport handshake (see [Appendix A](#appendix-a)).
```

Comment:
SPKI is "Simple public key infrastructure"? I understand what the sentence means, but not sure how to read it.. Should it be "SPKI certificate"?

Reply:
I mean here SPKI block, that is not including line breaks and header/footer

Reply:
updated